src/v/redpanda/admin_server.cc

```diff
@@ -1006,6 +1024,12 @@ void admin_server::register_security_routes() {
             = co_await _controller->get_security_frontend().local().delete_user(
               user, model::timeout_clock::now() + 5s);
           vlog(logger.debug, "Deleting user {}:{}", err, err.message());
```

Comment:
Is it worth adding the username here, too?

Reply:
Good point -- done.